swatch-system-conduit/src/main/spec/internal-organizations-sync-api-spec.yaml

```diff
@@ -0,0 +1,314 @@
+  version: 1.0.0
+paths:
+  /internal/rpc/syncOrg:
+    description: "Trigger a sync for a given Org ID"
+    post:
+      summary: "Sync organization for given org_id."
+      requestBody:
+        $ref: "#/components/requestBodies/OrgSyncRequestBody"
+      operationId: syncOrg
+      tags:
+        - internalOrganizations
+      responses:
+        '200':
+          description: "The request for syncing organization has completed successfully."
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: "#/components/schemas/DefaultResponse"
+        '400':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/BadRequest"
+        '403':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/Forbidden"
+        '404':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/ResourceNotFound"
+        '500':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/InternalServerError"
+  /internal/rpc/syncAllOrgs:
+    description: "Trigger a sync for all Organizations"
+    post:
+      summary: "Sync all organizations."
+      operationId: syncFullOrgList
+      tags:
+        - internalOrganizations
+      responses:
+        '202':
+          description: "The request for syncing all organizations is successfully running."
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: "#/components/schemas/DefaultResponse"
+        '400':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/BadRequest"
+        '403':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/Forbidden"
+        '404':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/ResourceNotFound"
+        '500':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/InternalServerError"
+  /internal/organizations-sync-list:
+    description: "Initiate altering database sync list by either adding or removing a given list of Org IDs"
+    parameters:
+      - name: org_ids
+        in: query
+        required: true
+        schema:
+          type: string
+        description: "Comma separated list of org_ids. (Ex. 123,456,789)"
+    post:
+      summary: "Add a given list of organizations to the database sync list."
+      operationId: addOrgsToSyncList
+      tags:
+        - internalOrganizations
+      responses:
+        '200':
+          description: "The request for adding organizations to the sync list has completed successfully."
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: "#/components/schemas/DefaultResponse"
+        '400':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/BadRequest"
+        '403':
+          $ref: "../../../../spec/error-responses.yaml#/$defs/Forbidden"
+        '404':
```

Comment:
Not sure this one would actually return a 404 either since its adding to a global list.